css/css-animations/parsing/animation-name-valid.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Animations Level 1: parsing animation-name with valid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://drafts.csswg.org/css-animations/#propdef-animation-name">
+<meta name="assert" content="animation-name supports the full grammar '[ none | <keyframes-name> ]#'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_valid_value("animation-name", 'none');
+test_valid_value("animation-name", 'foo');
+test_valid_value("animation-name", 'both');
+test_valid_value("animation-name", 'ease-in');
+test_valid_value("animation-name", 'infinite');
+test_valid_value("animation-name", 'paused');
+test_valid_value("animation-name", 'first, second, third');
```

Comment:
Can we add some tests for "strings"?
And escaped spaces in custom idents?
And otherwise-forbidden CSS-wide keywords which are allowed as strings, e.g. "initial"?
You can see some examples here: https://hg.mozilla.org/mozilla-central/rev/f41a419197de

Reply:
Tests added. They pass in Firefox.